06_modules_lab/02_bonus.py

```diff
@@ -1,10 +1,12 @@
-""" Write a program that reads 2 numbers from sys.argv
-and prints their sum.
-Bonus: Print error messages for invalid inputs.
-
-To print error messages we'll have to use a concept not yet learned in the
-course, and which will only be presented later: Exceptions.
-We'll tap into python's error handling and change its default
-error message to something more meaningful.
-"""
-
+""" 
+Print sum of two input  + check inputs
+"""
+#Week01_Exercise02_Assignment02_Bonus
+import sys
+if len(sys.argv) != 3: print 'Syntax Error. Syntax is:\npython %s <integer_value_1> <integer_value_2>' % sys.argv[0]
+else:
+    try:
+        x,y = int(sys.argv[1]),int(sys.argv[2])
+        print 'Sum of %d and %d is %d' %(x,y,x+y)
+    except ValueError:
+        print 'Syntax Error. Syntax is:\npython %s <integer_value_1> <integer_value_2>' % sys.argv[0]

```

Comment:
Fantastic! But since you already have try/except no need for the outer if. Just put everything inside the try/except block